aplicaciones/academico/serializers.py

```
@@ -4,6 +4,9 @@
 from aplicaciones.institucion.models import Aula
 from aplicaciones.personal.models import Profesor
 
+from rest_framework.serializers import ModelSerializer, CharField
```

Ruff (0.11.9)

`rest_framework.serializers.ModelSerializer` imported but unused

Remove unused import

(F401)